Preprocessing/pyproject.toml

```diff
@@ -10,6 +10,7 @@ python = "^3.8"
 pandas = "^1.3.4"
 simplejson = "^3.17.6"
 xlrd = "^2.0.1"
+openpyxl = "^3.0.10"
```

Comment:
@sbancal Was it necessary to keep the xls format for the DATA_Template ? was is possible to convert to csv ?

Reply:
Okay @charlottegiseleweil explained to me why. Maybe add a description on the process in the readme/ I could find where openpyxl where used.

Reply:
Researchers provide XLS(X) files, so we need to be able to parse it.
Not sure I understand ... would you like that we document in the README the reason for every dependency ? Or just that one ? Why this one ?

Reply:
@sbancal sorry, I was not clear enough. I meant something like: I didn't find any reference in the code to openpyxl, so I'm not sure which programm/script uses it. And about the README I meant that I thought it was a good idea to explain a bit more the process around the excel file: from what I understand we take the excel file and convert it into a json + csv ? Is that correct ?

Reply:
You're right. If I remember correctly, this library was missing when parsing XLS files with pandas ... but wasn't a direct dependency of pandas. That's the reason I installed it by hand.

Reply:
Yes, you're right. It's documented in the figure here https://github.com/EPFL-ENAC/CCFatiguePlatform#app-architecture where we see *TST Fatigue Test Data Directory (Excel + CSVs)* --> *Preprocessing* --> *TST Fatigue Test Data CSV* and *TST Fatigue Experiment JSON*.